packages/collector/test/tracing/database/redis/test.js

```diff
@@ -235,6 +237,13 @@ const globalAgent = require('../../../globalAgent');
                       ]);
 
                       verifyHttpExit(controls, spans, writeEntrySpan);
+
+                      // TODO: not sure what is the difference in span length with setupType
+                      if (setupType === 'cluster') {
+                        expect(spans.length).to.be.eql(2);
```

Comment:
Why the difference in span length for cluster and standalone?

Reply:
The PR only adds span lengths for the "current state".

Reply:
We have a similar behavior for `ioredis`.

Reply:
Extended https://jsw.ibm.com/browse/INSTA-14540

Reply:
I see, the initial comment was confusing.